extract.js

```diff
@@ -38,7 +38,10 @@ const extractor = new Extractor({
   attributes: finalAttrs
 })
 
-const vueFiles = glob.sync(`${srcFolder}/**/*.vue`)
+const getFileArrays = (fileExtension = 'vue') => srcFolder.map(src => glob.sync(`${src}/**/*.${fileExtension}`))
+const flattenFiles = (fileExtension = 'vue') => [].concat.apply([], getFileArrays())
```

Comment:
Is this function necessary? 

Because it doesn't make use of `fileExtension` and is creating an additional array from the results of `getFileArrays`

Perhaps we can just call:

```
const vueFiles = getFiles()
```

```
const jsFiles = getFiles('js').concat(vueFiles)
```

Reply:
glob.sync generates an array... so i'll have a multiple array... that's will be generate an erro.
but i fix that using a reduce function to help that
```
const syncFiles = (src, fileExtension = 'vue') => glob.sync(`${src}/**/*.${fileExtension}`)

const getFiles = (fileExtension = 'vue') => srcFolder.reduce((files, src) => [...files, ...syncFiles(src, fileExtension)], [])

const vueFiles = Array.isArray(srcFolder) ? getFiles() : glob.sync(srcFolder)
```
```
// note: vue files contain js code too
  const jsFiles = Array.isArray(srcFolder) ? getFiles('js') : syncFiles(srcFolder, 'js')
  const files = [...jsFiles, ...vueFiles]
```
what do you think about that?